src/maker/api.rs

```diff
@@ -510,9 +531,15 @@ pub fn check_for_broadcasted_contracts(maker: Arc<Maker>) -> Result<(), MakerErr
                             "[{}] Spawning recovery thread after seeing contracts in mempool",
                             maker.config.port
                         );
-                        let handle = std::thread::spawn(move || {
-                            recover_from_swap(maker_clone, outgoings, incomings).unwrap();
-                        });
+                        let handle = std::thread::Builder::new()
```

Comment:
nit: 
- Import `std::thread` at the start.
then:
``` suggestion 
                        let handle = thread::Builder::new()
```
and also make similar  nits in `src/maker/api` only.

Reply:
> and also make similar nits in src/maker/api only.
There are many places inside `src/maker/api `    where we are still using 
`std::thread::` even though `thread` is already  imported at the start.